app/src/main/java/com/nilhcem/blenamebadge/ui/fragments/main_text/MainTextDrawableFragment.kt

```diff
@@ -81,7 +81,7 @@ class MainTextDrawableFragment : BaseFragment(), MainTextDrawableNavigator {
             textToSend,
             flash.isChecked,
             marquee.isChecked,
-            Speed.values()[speed.selectedItemPosition],
+            Speed.values()[speed.value.minus(1)],
```

Comment:
Since 1 is a constant value being used in multiple places, declare it as a `private val` and then use it.

Reply:
We usually extract index from length by reducing 1. Are you sure we need to make a const for it

Reply:
Yeah well, we always do `length - 1` for the index. It's okay then.

Reply:
Great xD